### DISCUSSION TITLE
File upload problem

### DISCUSSION THREAD

**Discussion by lzb-libra** [requarks/wiki#6604] [Error / Bug Report]
When I create a page on wiki.js, I need to insert a picture into the article. I click the Insert File button on the left, which will prompt me that there is an exception.
DB：mysql
![QQ截图20230728145910](https://github.com/requarks/wiki/assets/63389341/c022ed5d-0715-4e7b-89b6-98e6598c0965)

**Comment by NGPixel** [comment]
Your MySQL version is outdated. You need 8.0 or later.

**Comment by lzb-libra** [comment]
Thank you for your advice. I'll try it first
